features/cookies.yml

```
@@ -1,8 +1,10 @@
-draft_date: 2024-10-21
-name: HTTP State Management Mechanism
-description: TODO
+name: Cookies
+description: Cookies store limited amounts of data and remember state information.
```

I think this is a good start. Maybe we can call out the actual entry point into this (I didn't mention `document.cookie`, though we could) and elaborating on where the state resides?

```suggestion
description: The `Set-Cookie` HTTP header stores limited amounts of data that persist across request and response, creating shared state between the browser and server.
```